DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/HostApp/RevitDocumentStore.cs

```diff
@@ -66,40 +69,39 @@ private void OnViewActivated(object sender, ViewActivatedEventArgs e)
       return;
     }
 
-    if (e.PreviousActiveView?.Document != null)
-    {
-      WriteToFileWithDoc(e.PreviousActiveView.Document);
-    }
-
     IsDocumentInit = true;
-    ReadFromFile();
-    OnDocumentChanged();
+    _idleManager.SubscribeToIdle(() =>
+    {
+      ReadFromFile();
+      OnDocumentChanged();
+    });
   }
 
   private void WriteToFileWithDoc(Document doc)
```

Comment:
Seems like this function is useless anymore. It was there before bc we were saving on doc switch with previous doc